gui/system/forms.py

```diff
@@ -2853,6 +2853,59 @@ class Meta:
         model = models.CertificateAuthority
 
 
+class CertificateAuthoritySignCSRForm(ModelForm):
+    cert_CSRs = forms.ChoiceField(
+        choices=[]
+    )
+
+    cert_name = forms.CharField(
+        label=models.Certificate._meta.get_field('cert_name').verbose_name,
+        required=True,
+        help_text=models.Certificate._meta.get_field('cert_name').help_text
+    )
```

Comment:
Why do you need the redefine cert_name here?

Reply:
I'm not sure what do you mean exactly? Why do I need to change the existing name of the certificate?

Reply:
No, there is a cert_name field in the model form already, what is the purpose of redefining it again, is what I am asking